sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -928,9 +890,7 @@
     },
     "/acr/v1/{name}/_manifests/{digest}": {
       "get": {
-        "tags": [
-          "Repository"
-        ],
+        "tags": ["Repository"],
         "description": "Get manifest attributes",
         "operationId": "ContainerRegistryRepository_GetManifestAttributes",
```

Comment:
Would it be possible to rename this method `GetRegistryArtifactProperties()` since it now returns a `RegistryArtifactsProperties` type?

Reply:
Done